piccolo_admin/example.py

```diff
@@ -180,6 +189,22 @@ async def booking_endpoint(request, data):
     return "Booking complete"
 
 
+@dataclass
+class LocalMediaHandler:
```

Comment:
I would make this a standard Python class, and have it inherit from a base class, called something like `MediaHandler`.

```
class MediaHandler(abc.ABC):
    @abc.abstract_method
    def upload(self, request, file):
        pass
```

Then it's more obvious what a user needs to implement if they create their own handler.